SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -205,7 +205,7 @@ void FSpatialGDKEditorToolbarModule::MapActions(TSharedPtr<class FUICommandList>
 
 	InPluginCommands->MapAction(
 		FSpatialGDKEditorToolbarCommands::Get().StartCloudSpatialDeployment,
```

Comment:
We should rename these commands to match the action as well.

Reply:
So I think StartCloudSpatialDeployment is an apt name. I've taken the time to cleanup some other names.